### PR TITLE
Style view petition

### PR DESCRIPTION
**Pull request by alexander-bobin**
This is a big one unfortunately. Pay most attention to the first and last commit.

There's a load of image inclusion and scss copy and pastes which you can ignore.

:8ball:

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,8 @@ public/cucumber
 public/index.html
 public/javascripts/all.js
 public/api/**
+public/assets/application-*.css
+public/assets/.sprockets-*.json
```
Comment (pixeltrix) [inline .gitignore]: We should just ignore the whole assets directory

File: .gitignore [hunk 2]
```diff
@@ -19,3 +21,7 @@ coverage
 */.DS_Store
 */*/.DS_Store
 .idea
+Gruntfile.js
+package.json
+node_modules
+.ruby-version
```
Comment (pixeltrix) [inline .gitignore]: Are we building the assets with Grunt? Why are these here?

**Comment by pixeltrix** [comment]
Why the``@javascript` for email confirmations to work? Isn't it the fact that we need to run the mail delivery job?

File: .gitignore (hunk 2 above)
Comment (alexander-bobin) [inline .gitignore]: I am using grunt locally for LiveReload

File: .gitignore (hunk 1 above)
Comment (alexander-bobin) [inline .gitignore]: Makes sense. Will change

**Comment by alexander-bobin** [comment]
It is because the steps involve interacting with a `detail` and `summary` node. This is a new HTML5 pattern used by govuk. Murray tells me the `@javascript` causes the use of a real(er?) browser which seems to handle these elements better. It doesn't seem to work without the `@javascript`

**Comment by otlaitil** [comment]
Looks good!

**Comment by mybits** [comment]
LGTM :+1:
